overlays/kuadrant/kustomization.yml

```diff
@@ -3,11 +3,34 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-  - ../../base/go-httpbin/
   - ../../base/mockserver/
   - ../../base/jaeger/
 
 images:
   - name: quay.io/rh_integration/go-httpbin
     newName: docker.io/mccutchen/go-httpbin
     newTag: latest
+
+patches:
+- patch: |-
+    $patch: delete
+    apiVersion: route.openshift.io/v1
+    kind: Route
+    metadata:
+      name: jaeger-query
```

Comment:
Why have some patches are inlined and some are in separate file?

Reply:
hmm, theoretically I could move all of the remove patches into one patch file

Reply:
It seems you cannot specify multiple delete patches in one file, meaning I would have to create file for each delete, since it is used once I do not think it is a good idea